inc/saiswitch.h

```diff
@@ -483,6 +534,7 @@ typedef sai_status_t (*sai_initialize_switch_fn)(
     _In_ sai_switch_profile_id_t profile_id,
     _In_reads_z_(SAI_MAX_HARDWARE_ID_LEN) char* switch_hardware_id,
     _In_reads_opt_z_(SAI_MAX_FIRMWARE_PATH_NAME_LEN) char* firmware_path_name,
+    _In_reads_opt_z_(SAI_MAX_PLATFORM_INFO_PATH_NAME_LEN) char* platform_info_path_name,
```

Comment:
instead of add a parameter to the init_switch_fn, can we use a K/V to provide the platform specifi file, e.g., INIT_CONFIG_FILE=platform_info_file?